src/app/components/Disclaimer/index.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import { ServiceContextProvider } from '../../contexts/ServiceContext';
+import { ToggleContextProvider } from '../../contexts/ToggleContext';
+import ThemeProvider from '../ThemeProvider';
+import { StoryProps } from '../../models/types/storybook';
+
+import metadata from './metadata.json';
+import DisclaimerComponent from '.';
+
+const Component = ({ service, variant }: StoryProps) => {
+  return (
+    <ToggleContextProvider
+      toggles={{
+        disclaimer: {
+          enabled: true,
+        },
+      }}
+    >
+      <ServiceContextProvider service={service}>
+        <ThemeProvider service={service} variant={variant}>
+          <DisclaimerComponent />
+        </ThemeProvider>
+      </ServiceContextProvider>
+    </ToggleContextProvider>
+  );
+};
+
+export default {
+  title: 'Components/Disclaimer',
+  Component,
+  parameters: {
+    metadata,
+    chromatic: { disable: true },
```

Comment:
Do we mean to omit the disclaimer component from the Chromatic (visual) snapshots?

Reply:
Ah no that was just copied across from the legacy component